getAlby/lightning-browser-extension
# feat: add support for viewing sats balance in locale currency and setting switch

#### Link this PR to an issue
Fixes #597 

#### Type of change (Remove other not matching type)
- New feature (non-breaking change which adds functionality)

#### Describe the changes you have made in this PR -

- add settings configuration for selecting fiat currency to display in app
- display fiat value of sats in selected currency

#### Screenshots of the changes (If any) -
<img width="398" alt="Screen Shot 2022-05-15 at 15 58 20" src="https://user-images.githubusercontent.com/104893211/168463093-fc225733-a592-4f86-8aee-6406d96ab5d2.png">

<img width="1556" alt="Screen Shot 2022-05-15 at 15 59 42" src="https://user-images.githubusercontent.com/104893211/168463145-fb884175-230c-411e-8107-621a9f46958c.png">

<img width="1556" alt="Screen Shot 2022-05-17 at 15 53 37" src="https://user-images.githubusercontent.com/104893211/168761331-191bc762-ebba-4cb5-aefe-65df058a2ce2.png">
<img width="412" alt="Screen Shot 2022-05-17 at 15 51 47" src="https://user-images.githubusercontent.com/104893211/168761344-3b5a70c1-d333-46ad-aef7-d11cc3d1bfd4.png">


#### How Has This Been Tested?

- Go to settings page
- Select your currency
- See the fiat value updated next to your sats

#### Checklist:

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)
